backend/services/syson-diagram-services/src/main/java/org/eclipse/syson/diagram/services/DiagramMutationElementService.java

```diff
@@ -72,22 +94,35 @@ public class DiagramMutationElementService {
 
     private final ModelMutationElementService modelMutationElementService;
 
+    private final IFeedbackMessageService feedbackMessageService;
+
+    private final MetamodelElementQueryService metamodelElementQueryService;
+
+    private final ModelQueryElementService modelQueryElementService;
+
+    private final IReadOnlyObjectPredicate readOnlyObjectPredicate;
+
     private final MetamodelMutationElementService metamodelMutationElementService;
 
     private final UtilService utilService;
 
     private final NodeDescriptionService nodeDescriptionService;
 
-    public DiagramMutationElementService(IIdentityService identityService, IObjectSearchService objectSearchService, IRepresentationDescriptionSearchService representationDescriptionSearchService,
+    public DiagramMutationElementService(SiriusWebCoreServices coreServices,
```

Comment:
Please try to order the services in parameters list:
1. what comes from SW
2. what comes from SysON